src/cljs/main/broadfcui/page/method_repo/method/common.cljs

```diff
@@ -38,41 +37,78 @@
-              :filter-bar {:inner {:width 300}}}}])
+(react/defc ConfigTable
+  {:refresh
+   (fn [{:keys [props state]}]
+     (swap! state dissoc :associated-configs :compatible-configs :resolved-configs :configs-error)
+     (endpoints/call-ajax-orch
+      {:endpoint (endpoints/get-agora-method-configs (:method-id props))
+       :on-done (net/handle-ajax-response
+                 (fn [{:keys [success? parsed-response]}]
+                   (if success?
+                     (swap! state assoc :associated-configs parsed-response)
+                     (swap! state assoc :configs-error (:message parsed-response)))))})
+     (endpoints/call-ajax-orch
+      {:endpoint (endpoints/get-agora-compatible-configs (conj (:method-id props) (select-keys props [:snapshot-id])))
+       :on-done (net/handle-ajax-response
+                 (fn [{:keys [success? parsed-response]}]
+                   (if success?
+                     (swap! state assoc :compatible-configs (set parsed-response))
+                     (swap! state assoc :configs-error (:message parsed-response)))))}))
+   :render
+   (fn [{:keys [props state]}]
+     (let [{:keys [style make-config-link-props]} props
+           {:keys [resolved-configs configs-error]} @state]
+       (cond configs-error
+             [:div {:style {:textAlign "center" :color (:state-exception style/colors)}}
+              "Error loading configs: " configs-error]
 
+             (not resolved-configs)
+             [:div {:style {:textAlign "center" :padding "1rem"}}
+              [comps/Spinner {:text "Loading configs..."}]]
 
-(defn render-post-export-dialog [{:keys [workspace-id config-id dismiss]}]
-  [modals/OKCancelForm
-   {:header "Export successful"
-    :content "Would you like to go to the edit page now?"
-    :cancel-text "No, stay here"
-    :dismiss dismiss
-    :ok-button
-    {:text "Yes"
-     :onClick #(mc-sync/flag-synchronization)
-     :href (nav/get-link :workspace-method-config workspace-id config-id)}}])
+             :else
+             [Table
+              {:data resolved-configs
+               :body {:empty-message "You don't have access to any published configurations for this method."
+                      :style (utils/deep-merge table-style/table-light
+                                               {:table {:backgroundColor "white"}}
+                                               style)
+                      :behavior {:reorderable-columns? false}
+                      :columns [{:id "compatible?" :initial-width 30 :resizable? false :sortable? false
+                                 :column-data :compatible?
+                                 :as-text (fn [c] (if c "Compatible" "Incompatible"))
+                                 :render (fn [c] (if c (icons/render-icon {:style {:color (:state-success style/colors)}} :done)
+                                                       (icons/render-icon {:style {:color (:state-warning style/colors)}} :warning)))}
```

Comment:
hmmm ... maybe use font-awesome's `exclamation-circle` here instead?

In the configs list inside a workspace, we use `exclamation-triangle` to indicate configs with redacted method snapshots. In the near future, we'll also have configs with redacted method snapshots inside agora. We should use consistent icons for that state, so it would be nice to have a circle icon here ... even though that's inconsistent with other warnings elsewhere.